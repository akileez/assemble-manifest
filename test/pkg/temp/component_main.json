{
  "name": "assemble-manifest",
  "version": "0.1.3",
  "main": [
    "a.js"
  ],
  "dependencies": {
    "graceful-fs": "~1.2.0",
    "grunt": "~0.4.1",
    "grunt-contrib-clean": "~0.4.1",
    "grunt-contrib-jshint": "~0.4.3",
    "grunt-contrib-nodeunit": "~0.1.2",
    "js-yaml": "~2.0.4",
    "lodash": "~1.2.0",
    "minimatch": "~0.2.12",
    "to": "~0.2.9"
  },
  "contributors": [
    {
      "name": "Brian Woodward",
      "url": "https://github.com/doowb"
    },
    {
      "name": "Jon Schlinkert",
      "url": "https://github.com/jonschlinkert"
    }
  ]
}