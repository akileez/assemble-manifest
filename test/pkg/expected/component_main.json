{
  "name": "my_component",
  "version": "0.1.0",
  "main": [
    "a.js"
  ],
  "dependencies": {
    "jquery": "1.9.2"
  }
}